man/systemd-getty-generator.xml

```diff
@@ -53,6 +53,38 @@
     Elsewhere)</ulink>.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Kernel Command Line</title>
+
+    <para><filename>systemd-getty-generator</filename> understands the following kernel command line
+    parameters:</para>
+
+    <variablelist class='kernel-commandline-options'>
+      <varlistentry>
+        <term><varname>systemd.getty_auto</varname></term>
+        <term><varname>rd.systemd.getty_auto</varname></term>
+
+        <listitem><para>Those options take an optional boolean argument, and default to yes.
+        The generator is enabled by default, and a negative value may be used to disable it.
```

Comment:
negative → false?

(i.e. most people would probably set =1 and =0 as values, but if you write negative people might misunderstand tht this means =-1...)